incubator/kafka/README.md

```diff
@@ -57,7 +57,7 @@ following configurable parameters:
 | Parameter                                      | Description                                                                                                                                                              | Default                                                            |
 |------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|
 | `image`                                        | Kafka Container image name                                                                                                                                               | `confluentinc/cp-kafka`                                            |
-| `imageTag`                                     | Kafka Container image tag                                                                                                                                                | `4.0.0`                                                            |
+| `imageTag`                                     | Kafka Container image tag                                                                                                                                                | `4.1.2-2`                                                            |
```

Comment:
Chart.yaml needs versionbump too, please also update the appVersion as well.

Reply:
Oh, sure, thanks for the notice